### ISSUE TITLE
Atom crashed: Attempting to call a function in a renderer window that has been closed or released

### ISSUE THREAD

**Issue by loibpet** [atom/atom#19161]
### Prerequisites

...

### Description

Atom crashed on Linux

Error Message:

Attempting to call a function in a renderer window that has been closed or released.
Function provided here: worker.js:79:22
Remote event names: destroyed, crashed

### Steps to Reproduce

1. Start atom
2. Tring to change the window => atom crashed

### Versions

Running on Linux

Atom    : 1.36.0
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3

### Additional Information

Problem was observerd only once so far and did occur after the first start after update to 1.36.0

**Comment by rsese** [comment]
Thanks for the report!

> Problem was observerd only once so far and did occur after the first start after update to 1.36.0

Thanks for these details, I wonder if the problem was specific to updating versions?  In any case, we'll close for now since the crash isn't reproducible but we can always reopen if you start seeing this again.

**Comment by gusmmm** [comment]
I have the same problem - after installing atom 1.36.0 on linux mint 19.1, I get the error message:
/usr/bin/atom: line 181: /home/gustavo/.atom/nohup.out: Permission denied
Attempting to call a function in a renderer window that has been closed or released.
Function provided here: worker.js:79:22
Remote event names: destroyed, crashed
Any suggestion?

**Comment by mellamanjorge** [comment]
Same here. Atom won't start on Linux Mint 19.1 when called from the GUI, without any error message. This is reproducible. 
When starting from console I get the above mentioned error message. Reproducible.
I can start atom from the console using sudo for some reason. 
Please reopen the bug, I'll be happy to help track it down. I'm not a developer, though, so someone has to walk me through it.

**Comment by gusmmm** [comment]
> Same here. Atom 1.36.1 won't start on Linux Mint 19.1 when called from the GUI, without any error message. This is reproducible.
> When starting from console I get the above mentioned error message. Reproducible.
> I can start atom from the console using sudo for some reason.
> Please reopen the bug, I'll be happy to help track it down. I'm not a developer, though, so someone has to walk me through it.

So I am not alone in the universe. Great tip - I am also able to open it without any problem using the terminal and sudo. 
I am not using atom because of this, which could be a good - I am learning new stuff using VCcode and pycharm - thing or a bad - not using atom - thing.
Thanks for caring.

**Comment by rsese** [comment]
@gusmmm @mellamanjorge - if one of you can open a new issue with the template filled out we'll take a look (the original report mentioned it's not reproducible for them so there seems to be a difference).

**Comment by Sharif55** [comment]
ATOM crashes if apply ctrl+c to get  command prompt from gitbash atom crashed and following error shows
.... AppData\Local\atom\app-1.37.0\resources\app.asar\node_modules\github\lib\worker.js:79:22
Remote event names: destroyed, crashed
